app/models/ems_event.rb

```diff
@@ -78,6 +78,7 @@ def self.add(ems_id, event_hash)
     process_availability_zone_in_event!(event_hash)
     process_cluster_in_event!(event_hash)
     process_container_entities_in_event!(event_hash)
+    process_storage_in_event!(event_hash)
```

Comment:
Prefer 
```suggestion
    process_physical_storage_in_event!(event_hash)
```

Since `Storage` is a different model